components/feed/FeedItem.tsx

```diff
@@ -42,23 +42,6 @@ const FeedItem = ({
       <p>{data.content}</p>
       <div className="flex flex-row justify-between items-center pt-30 pb-5 pl-15">
         <div className="flex flex-row justify-start items-center">
```

Comment:
Probably want to remove the this flex-row div and the div around the like button since they're unused now.